scripts/secrets/update.py

```diff
@@ -102,7 +104,11 @@ def represent_none(self, data):
 def sealed_secret_cert_path(staging: bool) -> str:
     env = "staging" if staging else "production"
     default_cert = (
-        Path(__file__).parent.parent / "k8s" / env / "sealed-secrets" / "cert.pem"
+        Path(__file__).parent.parent.parent
```

Comment:
Something I learned about `pathlib` recently because I've found myself appending several `.parent`s to Path objects many times before :smile: 
```suggestion
        Path(__file__).parents[2]
```
Not that this is a big deal in this specific case, since three chained `.parents` is still readable imo, but thought i would share nonetheless

Reply:
For some reason when it comes to usability I usually just do several `.parent` calls, but you're right that using `parents` is the better approach here.